book/listings/todo_app/1/main.rs

```diff
@@ -20,39 +17,16 @@ fn main() {
         .application_id("org.gtk.Todo")
```

Comment:
Do other examples also use `org.gtk`? I thought maybe `org.gtk-rs` would be more appropriate.

Reply:
> I thought maybe org.gtk-rs would be more appropriate.

Agreed